JS/scripts.js

```diff
@@ -0,0 +1,76 @@
+// Loading html file dymanically
+const desktopSpeakers = document.querySelector('.desktopSpeakers');
+
+const speakers = [
+  {
+    topic: 'Featured Speakers',
+    name: 'Yochai Benkler',
+    image: 'images/speaker_01.png',
+    roles: 'Berkman Professor of Entre preneurial Legal Studies at Harvard Law School',
+    achievements: 'Benkler studies commons-based peer production, Networks in 2006.',
+
+  },
+  {
+    topic: 'Featured Speakers',
+    name: 'Sohyeong Noh',
+    image: 'images/speaker_03.png',
+    roles: 'Director of Art Centre Nabi and a board member of CC Korea',
+    achievements: 'As the main venue of new media art production in Korea, Nabi promotes cross-disciplinary collaboration, humanities, and the arts',
+
+  },
+  {
+    topic: 'Featured Speakers',
+    name: 'Sohyeong Noh',
+    image: 'images/speaker_03.png',
+    roles: 'Director of Art Centre Nabi and a board member of CC Korea',
+    achievements: 'As the main venue of new media art production in Korea, Nabi promotes cross-disciplinary collaboration, humanities, and the arts',
+
+  },
+  {
+    topic: 'Featured Speakers',
+    name: 'Sohyeong Noh',
+    image: 'images/speaker_01.png',
+    roles: 'Director of Art Centre Nabi and a board member of CC Korea',
+    achievements: 'As the main venue of new media art production in Korea, Nabi promotes cross-disciplinary collaboration, humanities, and the arts',
+
+  },
+  {
+    topic: 'Featured Speakers',
+    name: 'Sohyeong Noh',
+    image: 'images/speaker_01.png',
+    roles: 'Director of Art Centre Nabi and a board member of CC Korea',
+    achievements: 'As the main venue of new media art production in Korea, Nabi promotes cross-disciplinary collaboration, humanities, and the arts',
+
+  },
+  {
+    topic: 'Featured Speakers',
+    name: 'Sohyeong Noh',
+    image: 'images/speaker_03.png',
+    roles: 'Director of Art Centre Nabi and a board member of CC Korea',
+    achievements: 'As the main venue of new media art production in Korea, Nabi promotes cross-disciplinary collaboration, humanities, and the arts',
+    more: 'MORE',
+  },
+];
+
+// Adding cards dynamically
+for (let i = 0; i < speakers.length; i += 1) {
+  desktopSpeakers.innerHTML += `
+    <div class="speakersContainer">
+    <div class="speaker1">
+        <div class="speakersImage">
+            <img class="speakersImage1" src=${speakers[i].image} alt="Speaker One">
```

Comment:
- [ Required ] Great job adding an `alt` text to each image of your speakers to make the web page accessible as requested by the previous reviewer, however, as currently implemented the images of all the speakers will have the same `alt` text. I suggest adding an `alt` key in the `speakers` array of objects so that each speaker image can have a distinct `alt` text.